quickstarts/install-helmchartrepo-ns.yaml

```diff
@@ -0,0 +1,95 @@
+  introduction: >-
+    Content in the Helm Chart Catalog is defined by **HelmChartRepository** and
+    **ProjectHelmChartRepository** CRs. Both CRs provide a URL and are used to
+    populate the available Helm Charts in the Developer Catalog. A default
+    **HelmChartRepository** is provided, called the **openshift-helm-charts**.
+
+    - **HelmChartRepository** CRs which are cluster scoped and defined by users
+    with the admin role
+
+    - **ProjectHelmChartRepository** CRs which are namespace-scoped and defined
+    by users with read/write access to the namespace
+  tags:
+    - helm
+  tasks:
+    - description: >-
+        Follow these steps to create a ProjectHelmChartRepository CR:
+
+        1. Click on the [perspective switcher]{{highlight
+        qs-perspective-switcher}} at the top of the navigation, and select
+        **Developer**.
+
+        1. In the navigation menu, click [Search]{{highlight qs-nav-search}}.
+
+        1. In the project selector, select the project to populate with the new
+        Helm Charts.
+
+        1. Click the **Resources dropdown** menu and select
+        **ProjectHelmChartRepository**.
+
+        1. Click **Create Project Helm Chart Repository** to bring up the YAML
+        editor.
+
+        1. Replace the contents of the YAML editor with:
+
+            ```
+            apiVersion: helm.openshift.io/v1beta1
+
+            kind: ProjectHelmChartRepository
+
+            metadata:
+
+              name: my-helm-chart-repo
+
+            spec:
+
+              connectionConfig:
+                url: 'https://raw.githubusercontent.com/Azure-Samples/helm-charts/master/docs'
+
+              name: My Helm Chart Repo
+
```

Comment:
Don't see a reason for these empty lines in the YAML example:

With this change it looks like this:

![image](https://user-images.githubusercontent.com/139310/150980506-aa2ebdf2-8635-4f88-a4d2-ae787b1dc087.png)

```suggestion
            apiVersion: helm.openshift.io/v1beta1
            kind: ProjectHelmChartRepository
            metadata:
              name: my-helm-chart-repo
            spec:
              connectionConfig:
                url: 'https://raw.githubusercontent.com/Azure-Samples/helm-charts/master/docs'
              name: My Helm Chart Repo
```

Did we add also an option to allow the user to copy the snippet? But I don't find any quickstart which uses this feature :shrug:

Reply:
Removed spaces.  We don't have snippets to date @jerolimov but maybe something to investigate for the future!